modules/va_facilities/README.yml

```diff
@@ -71,19 +71,37 @@ paths:
         services. Only facilities of type "health" and "benefits" may be filtered by available
         services.
 
+        Alternatively, one can retrieve multiple facilities by id using this endpoint by making a
+        request with a comma-separated list of ids like `?ids=id1,id1`. When requesting facilities
```

Comment:
it might be clearer in the readme to say `?ids=id1,id2` instead of `?ids=id1,id1`